apps/omg_watcher/lib/omg_watcher/api/in_flight_exit.ex

```diff
@@ -92,29 +94,22 @@ defmodule OMG.Watcher.API.InFlightExit do
   end
 
   defp find_input_data(tx) do
-    result =
-      tx
-      |> Transaction.get_inputs()
-      |> Enum.map(fn
-        utxo_pos ->
-          with {:ok, %{proof: proof, txbytes: txbytes}} <- API.Utxo.compose_utxo_exit(utxo_pos),
-               do: {proof, txbytes}
-      end)
-
-    result
-    |> Enum.any?(&match?({:error, :utxo_not_found}, &1))
-    |> case do
-      true -> {:error, :tx_for_input_not_found}
-      false -> {:ok, Enum.unzip(result)}
-    end
+    tx
+    |> Transaction.get_inputs()
+    |> Enum.reverse()
```

Comment:
Yeah, if they go together let's extract them to a new function that we can use in pipe there.
I can see now that "reverting outputs we gain not need to revert 3 resulting lists" - this could be a good comment to the new function actually.

Reply:
but this is almost exactly the function you're asking to add :). Hmmm maybe I'll try with some comment.